cmd/boost/provider_cmd.go

```diff
@@ -267,7 +286,8 @@ var retrievalAskCmd = &cli.Command{
 		var ask retrievalmarket.QueryResponse
 
 		if err := doRpc(ctx, s, &req, &ask); err != nil {
-			return fmt.Errorf("send retrieval-ask request rpc: %w", err)
+			augmentedError := fmt.Errorf("send retrieval-ask request rpc: %w", err)
+			return cmd.PrintError(augmentedError, outputInJson)
 		}
 
 		afmt.Printf("Status: %d\n", ask.Status)
```

Comment:
Why do we use the `afmt` special command here and not everywhere else?
Why don't we support `json` output for this command as well?